contracts/Safe.sol

```diff
@@ -145,37 +167,37 @@ contract Safe is
                     operation,
                     safeTxGas,
-                    // Payment info
+                    // Payment info:
                     baseGas,
                     gasPrice,
                     gasToken,
                     refundReceiver,
-                    // Signature info
+                    // Signature info:
                     signatures,
                     msg.sender
                 );
             }
         }
 
-        // We require some gas to emit the events (at least 2500) after the execution and some to perform code until the execution (500)
-        // We also include the 1/64 in the check that is not sent along with a call to counteract potential shortings because of EIP-150
-        // We use `<< 6` instead of `* 64` as SHR / SHL opcode only uses 3 gas, while DIV / MUL opcode uses 5 gas.
+        // We require some gas to emit the events (at least 2500) after the execution and some to perform code until the execution (500).
+        // We also include the 1/64 in the check that is not sent along with a call to counteract potential shortings because of EIP-150.
+        // We use `<< 6` instead of `* 64` as the `SHL` opcode only uses 3 gas, while the equivalent `MUL` opcode uses 5 gas.
         if (gasleft() < ((safeTxGas << 6) / 63).max(safeTxGas + 2500) + 500) revertWithError("GS010");
-        // Use scope here to limit variable lifetime and prevent `stack too deep` errors
+        // Use scope here to limit variable lifetime and prevent "stack too deep" errors.
         {
             uint256 gasUsed = gasleft();
-            // If the gasPrice is 0 we assume that nearly all available gas can be used (it is always more than safeTxGas)
-            // We only subtract 2500 (compared to the 3000 before) to ensure that the amount passed is still higher than safeTxGas
+            // If the `gasPrice` is 0 we assume that nearly all available gas can be used (it is always more than `safeTxGas`).
+            // We only subtract 2500 (compared to the 3000 before) to ensure that the amount passed is still higher than `safeTxGas`.
             success = execute(to, value, data, operation, gasPrice == 0 ? (gasleft() - 2500) : safeTxGas);
             gasUsed = gasUsed.sub(gasleft());
-            // If no safeTxGas and no gasPrice was set (e.g. both are 0), then the internal tx is required to be successful
-            // This makes it possible to use `estimateGas` without issues, as it searches for the minimum gas where the tx doesn't revert
+            // If no `safeTxGas` and no `gasPrice` was set (i.e. both are 0), then the internal transaction must be successful.
+            // This makes it possible to use `estimateGas` without issues, as it searches for the minimum gas where the transaction doesn't revert.
             if (!success && safeTxGas == 0 && gasPrice == 0) {
                 /* solhint-disable no-inline-assembly */
                 /// @solidity memory-safe-assembly
```

Comment:
```suggestion
                // @solidity memory-safe-assembly
```

Reply:
According to the Solidity docs, this needs to be `///`:

https://docs.soliditylang.org/en/v0.8.29/assembly.html#memory-safety